### DISCUSSION TITLE
Pigsty 有对应的 mysql 版本吗？

### DISCUSSION THREAD

**Discussion by ponponon** [pgsty/pigsty#349] [Q&A]
看到一篇介绍 Pigsty 的文章，https://mp.weixin.qq.com/s/lHp5RANMncx0VUmCHCwwWw

> https://github.com/Vonng/pigsty

我感觉这是个好东西，但是我现在用的都是 mysql，mysql 下面有类似 Pigsty 这套东西吗？

**Comment by Vonng** [comment]
we have alpha mysql support on 2.5.1. Will have monitoring support for MySQL 8.x in v3

**Reply by Songxwn** [reply]
强烈支持
